.kokoro/docker/Dockerfile

```diff
@@ -174,4 +174,11 @@ RUN useradd -d /h -u ${UID} ${USERNAME}
 # Allow nopasswd sudo
 RUN echo "${USERNAME} ALL=(ALL) NOPASSWD:ALL" >> /etc/sudoers
 
+# Setup DRIFT region tag parser
+ENV REGION_TAG_PARSER_DIR /region-tag-parser
+ENV PARSER_PATH /region-tag-parser/wizard-py/cli.py
+
+RUN git clone https://github.com/GoogleCloudPlatform/repo-automation-playground "$REGION_TAG_PARSER_DIR" --single-branch
```

Comment:
It will always fetch it from head in the docker build.

This will prevent you from "releasing" the tool at will.

We always build the docker image, but we're utilizing previously built image as the cache. `docker build` command can not detect new changes in `repo-automation-playground` repo, so `docker build` will just use the cache layer and we won't get the new changes.

Maybe you want to have commit hash at least?

Reply:
Good point - fixed.